script.js

```diff
@@ -127,9 +127,22 @@ function renderQuestion() {
 }
 
 function endGame() {
-  console.log('game ended');
+  document.querySelector('.quiz-container').remove();
+  var d = document.getElementById("game-container")
+  d.className='end-game';
+  var e = document.getElementById("game-score")
+  e.style.color='black';
+  e.innerHTML=renderendgameText();
+}
+function renderendgameText(){
+  return(
+    `<div>
+  <h4>GAME OVER!</h4>
```

Comment:
Instead of adding html via JS, you can put the html inside the html file only and add a is-hidden class on it by default.

When the game is over, remove the hidden class from the game-over section and put that class on the quiz section. 

You can define the `is-hidden` class as follows-

```css
.is-hidden {
  display: none;
}
```